zio-http/shared/src/main/scala/zio/http/Server.scala

```diff
@@ -578,3 +585,18 @@ object Server extends ServerPlatformSpecific {
 
   }
 }
+
+final case class ServerRuntimeConfig(
```

Comment:
We can't have a new class ever time we add a config. And while this is bin compatible, it is not source compatible.
It might be more work, but please make the other config a class instead of a case class and write all necessary methods by hand